src/java.base/share/classes/java/util/Deque.java

```diff
@@ -617,8 +617,13 @@
      * {@inheritDoc}
      *
      * @implSpec
-     * The implementation in this interface returns an instance of a reverse-ordered
-     * Deque that delegates its operations to this Deque.
+     * The implementation in this interface returns a reverse-ordered Deque
+     * view that delegates its operations to this Deque. Methods that are
+     * order-insensitive delegate to the corresponding method of this Deque.
+     * Methods that are order-sensitive delegate to the corresponding but
+     * inverse-ordered method of this Deque. For example, the {@code getFirst}
+     * method of the returned view delegates to the {@code getLast} method of
+     * this Deque.
```

Comment:
The description is general and using 'For example" avoids having to be specific about every operation.
The term "inverse-ordered" is new to this context and perhaps should be "reverse-ordered".
"inverse" does appear in a few other class descriptions for SequencedXXX but its relation to "reverse" may not be clear.

Reply:
Rather than describing how the methods delegate, I think we mostly want to say that 
order-sensitive operations are performed in reverse order.

I agree about, "For example", and I would keep the `getFirst` -> `getLast` part - that's a good, straighforward example.